admin_guide/diagnostics_tool.adoc

```diff
@@ -349,14 +360,14 @@ playbook. For example:
 openshift_disable_check=etcd_traffic,etcd_volume
 ----
 
-Alternatively, set any checks you want to disable as variables with
-`-e openshift_disable_check=<check1>,<check2>` when running the
-`ansible-playbook` command.
+Alternatively, set any checks to disable as variables with `-e
```

Comment:
Are these variables or parameter values? Do you mean this?

Alternatively, specify the names of checks to disable as values of the `openshift_disable_check` parameter, as shown:

 ----
 # ansible-playbook -e openshift_disable_check=<check1>,<check2>
 ----